core/frontend/src/tile/RealityModelTileTree.ts

```diff
@@ -863,30 +860,26 @@ export class RealityModelTileClient {
     }
   }
 
-  // SWB
-  // ###TODO temporary means of extracting the tileId and projectId from the given url
+  // ###TODO temporary means of extracting the tileId and iTwinId from the given url
   // This is the method that determines if the url refers to Reality Data stored on PW Context Share. If not then undefined is returned.
   // ###TODO This method should be replaced by realityDataServiceClient.getRealityDataIdFromUrl()
-  // SWB
-  // We obtain the projectId from URL but it should be used normally. The iModel context should be used everywhere: verify!
-  // SWB
+  // SWB What does context mean here?
+  // We obtain the iTwinId from URL but it should be used normally. The iModel context should be used everywhere: verify!

```

Comment:
Should "iModel context" be reworded?

Reply:
leave as-is for now. It's internal and isn't referencing an actual Connect Context so is fine.